global/apps/homepage/views.py

```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
 
 @cache_control(no_cache=True)
 def about(request):
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY')
+    if country_code is None:
+        country_code = request.META.get(
+            'GEOIP_COUNTRY_CODE',
+            '--',
+        )
+
+    # Fetch the list of contributors from the various missions. Use a
+    # set() because we only want to list each contributor once.
+    contributors = set()
+    for mission in list(Mission.Query(request.redis_conn)):
+        if not mission.incomplete:
+            contributors = contributors | set(mission.copy.get('cleaners', []))
```

Comment:
Why do we need to copy the mission here? We don't seem to be mutating the mission, and even if we were, we don't seem to be using the missions again after this loop.